MATLAB/Demos/d08_Algorithms03.m

```diff
@@ -63,6 +63,8 @@
  
 % use CGLS
 [imgCGLS, errL2CGLS]=CGLS(noise_projections,geo,angles,60);
+% use LSQR

```

Comment:
maybe (at a more advanced stage of our work) we can try to explain why use one or the other? Or at least their differences.

Reply:
Not sure when you should ever use CGLS! But could be persuaded... we can look into that.